cdap-ui/app/cdap/components/PipelineConfigurations/PipelineConfigurations.scss

```diff
@@ -0,0 +1,459 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+@import "../../styles/variables.scss";
+
+$text-color: $grey-01;
+$border-color: $grey-05;
+$active-bg-color: $grey-07;
+$input-box-focus-color: #66afe9;
+$input-label-color: $grey-03;
+$action-button-color: $grey-02;
+$modeless-bg-color: $grey-06;
+$modeless-header-bg-color: $grey-08;
+$modeless-width: 830px;
+$modeless-height: 496px;
+$side-panel-width: 170px;
+$regular-font: 14px;
+$secondary-btn-color: $brand-primary;
+
+body.theme-cdap {
+  &.state-hydrator {
+    .pipeline-configurations-content {
+      background: $modeless-bg-color;
+      position: fixed;
+      top: 95px;
+      overflow: auto;
+      right: 15px;
+      z-index: 999;
+      box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.5);
+      color: $text-color;
+      background-color: white;
```

Comment:
If we have finalized the modeless styling I guess we should probably extract this out to a mixin and use it everywhere for consistency.